tezos/api/src/environment.rs

```diff
@@ -27,9 +29,12 @@ use crypto::{base58::FromBase58CheckError, blake2b::Blake2bError};
 use tezos_messages::p2p::encoding::prelude::{BlockHeader, BlockHeaderBuilder};
 
 use crate::ffi::{GenesisChain, PatchContext, ProtocolOverrides};
+use crate::octez_config::OctezConfig;
 
 lazy_static! {
     pub static ref TEZOS_ENV: HashMap<TezosEnvironment, TezosEnvironmentConfiguration> = init();
```

Comment:
@tizoc 
I dont like this `lazy_static TEZOS_ENV` , I would like to remove it totally,
I guess, now we dont need it, only just for tests,

so I would like to remove this lazy_static,
and then we just need to fix this:
```
let env = crate::configuration::Environment::from_args();
    let tezos_env = environment::TEZOS_ENV
        .get(&env.tezos_network)
        .unwrap_or_else(|| {
            panic!(
                "No tezos environment version configured for: {:?}",
                env.tezos_network
            )
        });
```

we just need to add `tezos_env: TezosEnvironmentConfiguration` to 
```
pub struct Environment {
    ...
    pub tezos_network: TezosEnvironment,
    ...
```

and join it with `tezos_network`


so in that main function we just call:
```
let env = crate::configuration::Environment::from_args();
let tezos_env = env.tezoz_env()
```

so, that actual hashmap, we dont need it, we just do that `init` as a part of `configuration.rs` on startup and there we just handle that custom network from file, like you did

I guess, this solution/refactor will clean the stuff a lot

Reply:
@bkontur sounds good to me. Only issue is that right now I have to work on things of much higher priority, so I will not be able to get back to this for some time. It can be merged with another TODO + jira for the refactor, or otherwise left for later.